phpstan.neon

```diff
@@ -0,0 +1,3 @@
+parameters:
+    ignoreErrors:
+        - '#Property .* does not accept#'
```

Comment:
@felixfbecker I (or PHPStan) noticed that all of the required properties were nullable at the constructor. Is this a requirement for AdvancedJsonRpc somehow? or could we make them required?

Reply:
Good question, it might be a requirement for the RPC library since I don't remember if [JSON Mapper](https://github.com/cweiske/jsonmapper) would support passing those to the constructor (or maybe it creates the classes with reflection without calling the contructor?). You could try it out but it's probably not worth it. If it works I'm happy to change it though.